src/core/load-dropin.c

```diff
@@ -76,10 +80,17 @@ static int process_deps(Unit *u, UnitDependency dependency, const char *dir_suff
 
                 /* We don't treat this as an error, especially because we didn't check this for a
                  * long time. Nevertheless, we warn, because such mismatch can be mighty confusing. */
```

Comment:
The comment belongs to `unit_symlink_name_compatible` below, not here

Reply:
I'm not seeing that comment as being 'new', here? Is that really a blocker, or a prompt for a new PR?!

Reply:
Please move it down I mean, close to unit_symlink_name_compatible